### PR TITLE
[Rule Tuning] Update event.category for Azure rules

### LINKED ISSUE #333: [Rule Tuning] Update Azure rules to remove event.category
## Description
It appears that `event.category` in the Azure module has incorrect field mappings. We are seeing `event.category:Administrative`  for Activity logs and `event.category:AuditLogs` for Audit logs. According to the [documentation](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-category.html), neither appear to be valid values. Example issue with screenshot - https://github.com/elastic/detection-rules/issues/197.

`event.category` isn't truly necessary as we are using the `event.dataset` field in all rules. I've also opened a beats issue https://github.com/elastic/beats/issues/21259 to get this taken care of. I suggest updating all Azure rules for 7.10 to eliminate this unneeded field from the rules.

image

## Screenshots

image

image

### PR DESCRIPTION
**Pull request by bm11100**
## Issues
resolves #333 

## Summary
In beats version 7.9, `event.category` is no longer set - elastic/beats#19376

We need to eliminate this field from our applicable Azure rules so they fire correctly with an updated beat.

We are using the `event.dataset` field in all Azure rules so the category isn't necessary.

Also, see https://github.com/elastic/beats/issues/21190

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by threat-punter — APPROVED** [review]
🧉

File: rules/azure/persistence_user_added_as_owner_for_azure_service_principal.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ tags = ["Elastic", "Azure", "Continuous Monitoring", "SecOps", "Configuration Au
 type = "query"
 
 query = '''
-event.module:azure and event.dataset:azure.auditlogs and event.category:AuditLogs and azure.auditlogs.operation_name:"Add owner to service principal" and event.outcome:Success
+event.module:azure and event.dataset:azure.auditlogs and azure.auditlogs.operation_name:"Add owner to service principal" and event.outcome:Success
```
Comment (brokensound77) [inline rules/azure/persistence_user_added_as_owner_for_azure_service_principal.toml:32]: This looks good, but can you update the name to include `Azure`

**Review by brokensound77 — APPROVED** [review]
Just the one comment, otherwise LGTM 👍

File: rules/azure/persistence_user_added_as_owner_for_azure_service_principal.toml (hunk 1 above)
Comment (bm11100) [reply rules/azure/persistence_user_added_as_owner_for_azure_service_principal.toml:32]: nice, updated, verified all others have Azure in the name

**Pull request merged**